docs/build.ps1

```diff
@@ -1,22 +1,31 @@
+Write-Host "Branch: $env:APPVEYOR_REPO_BRANCH"
+Write-Host "PR Number: $env:APPVEYOR_PULL_REQUEST_NUMBER"
+Write-Host "Job Name: $env:APPVEYOR_JOB_NAME"
+$haveAccessToken = -not -not $env:git_access_token
+Write-Host "Have Access Token: $haveACcessToken"
+
 
 if($env:APPVEYOR_REPO_BRANCH -eq "master" -and -not $env:APPVEYOR_PULL_REQUEST_NUMBER -and $env:APPVEYOR_JOB_NAME -like "*Release*" -and $env:git_access_token){
-	$Deploy="true"
+	  $Deploy="true"
+    Write-Host "Deploying docs"
+} else {
+    Write-Host "Not deploying docs"
 }
 
 $docfxVersion = "2.33.0"
 & nuget install docfx.console -Version $docfxVersion -Source https://api.nuget.org/v3/index.json
 
 if($Deploy){
-	# Configuring git credentials
-	& git config --global credential.helper store
-	Add-Content "$env:USERPROFILE\.git-credentials" "https://$($env:git_access_token):x-oauth-basic@github.com`n"
+	  # Configuring git credentials
```

Comment:
I know that this is mostly from my frantic throwing this script together, but the whitespace seems off...